cmd/gosec/main.go

```diff
@@ -457,6 +467,14 @@ func main() {
 
 	reportInfo := gosec.NewReportInfo(issues, metrics, errors).WithVersion(Version)
 
+	// Call ai request to solve the issues
+	if *flagAiApiProvider != "" && *flagAiApiKey != "" {
+		err := autofix.GenerateSolution(*flagAiApiProvider, *flagAiApiKey, *flagAiEndpoint, issues)
+		if err != nil {
+			logger.Print(err)
```

Comment:
Why not using logger.Fatal?

Reply:
[Here](https://github.com/securego/gosec/pull/1177/#discussion_r1701421643) is the answer.

Reply:
Thanks

I would add a comment explaining it here then